workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetApp/components/SelectPaymentMethod/BillingAddress/BillingAddress.tsx

```diff
@@ -246,7 +246,7 @@ export function BillingAddress({
 
 						<div className="d-flex">
 							<ClayCheckbox
-								checked={billingAddress?.saveAddress}
+								checked={billingAddress?.saveAddress ?? false}
```

Comment:
Only change to use this validation

```suggestion
								checked={!!billingAddress?.saveAddress }
```